MultiSource/Benchmarks/Olden/treeadd/node.c

```diff
@@ -124,7 +124,7 @@ int TreeAdd (tree_t *t)
 #else
     int leftval;
     int rightval;
-    tree_t *tleft, *tright;
+    Tree tleft = NULL, tright = NULL;
```

Comment:
yes. I will. but there is some file I cannot find such as future-cell.h. So I thought that this ifdef will not be defined.